R/fotmob_stats.R

```diff
@@ -77,141 +70,84 @@
     league_name,
     league_id,
     team_or_player,
-    page_url,
     cached
 ) {
 
-  tables <- fotmob_get_league_tables(
```

Comment:
this whole function was a mess before because they had changed the way that we could access season ids and stat names several times over the past year. seems like they offer that data at the top-level league endpoint now, which makes things much simpler